src/main/java/com/exasol/adapter/dynamodb/documentfetcher/dynamodb/DynamodbQueryDocumentFetcher.java

```diff
@@ -57,12 +57,14 @@ private void addKeyConditionAndFilterExpression(final RemoteTableQuery<DynamodbN
             final DynamodbIndex mostRestrictedIndex) {
         final QueryPredicate<DynamodbNodeVisitor> selectionOnIndex = new DynamodbQuerySelectionFilter()
                 .filter(documentQuery.getSelection(), getIndexPropertyNameWhitelist(mostRestrictedIndex));
-        final DynamodbValueListBuilder valueListBuilder = new DynamodbValueListBuilder();
+        final DynamodbAttributeNamePlaceholderMapBuilder namePlaceholderBuilder = new DynamodbAttributeNamePlaceholderMapBuilder();
```

Comment:
unify the variable names for these classes somehow. They are  different in different places